resources/styles/variables/colors.less

```diff
@@ -35,20 +35,31 @@
 @exercise-question-uid-color: lighten(@tutor-neutral,20%);
 
 // TODO: Move book styles to react-components
-// NOTE: These must be kept in sync with the books in src/flux/book-appearance-codes.coffee
-@openstax-book-appearance-codes: default physics hs_physics biology principles_economics macro_economics college_physics micro_economics concepts_biology intro_sociology anatomy_physiology;
+// NOTE:
+//  * The 'biology' and 'physics' codes are deprecated, but are retained for older courses that may have them
+//  * These must be kept in sync with the books in src/flux/book-appearance-codes.coffee
+@openstax-book-appearance-codes: default physics hs_physics college_physics biology ap_biology concepts_biology college_biology principles_economics macro_economics micro_economics intro_sociology anatomy_physiology;
+
```

Comment:
yeah, that'd be great to do.  I do some crazy less looping over the appearance codes listed there.  Maybe we could make a `book-content.less` file or something?